component/alertrules.libsonnet

```diff
@@ -9,8 +9,8 @@ local elasticsearch = inv.parameters.openshift4_logging.components.elasticsearch
 local loki = inv.parameters.openshift4_logging.components.lokistack;
 
 
-local runbook(alertname) =
-  'https://hub.syn.tools/openshift4-logging/runbooks/%s.html' % alertname;
+local runbook(alertname) = 'https://hub.syn.tools/openshift4-logging/runbooks/%s.html' % alertname;
+local runbookLoki(alertname) = '' % alertname;
```

Comment:
This isn't used as far as I can see?

```suggestion
```

Reply:
I think its used with the prediction alerts of ES

Reply:
yes the `runbook()` function is used, but `runbookLoki()` isn't used as far as I can see.

Reply:
ah, missed 🤦